_utils/image_names_output.txt

```diff
@@ -0,0 +1,264 @@
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("pate12"),
+this
+),
+
+new CardObject(
+this,
+{ name: "farine01", url: "sprites/StoreScene/part1/rayon01_03_farine01.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("farine01"),
+this
+),
+
+new CardObject(
+this,
+{ name: "pate07", url: "sprites/StoreScene/part1/rayon01_03_pate07.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("pate07"),
+this
+),
+
+new CardObject(
+this,
+{ name: "sac03", url: "sprites/StoreScene/part1/rayon01_03_sac03.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("sac03"),
+this
+),
+
+new CardObject(
+this,
+{ name: "spaghetti02", url: "sprites/StoreScene/part1/rayon01_03_spaghetti02.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("spaghetti02"),
+this
+),
+
+new CardObject(
+this,
+{ name: "spaghetti03", url: "sprites/StoreScene/part1/rayon01_03_spaghetti03.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("spaghetti03"),
+this
+),
+
+new CardObject(
+this,
+{ name: "spaghetti01", url: "sprites/StoreScene/part1/rayon01_03_spaghetti01.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("spaghetti01"),
+this
+),
+
+new CardObject(
+this,
+{ name: "rayonnage", url: "sprites/StoreScene/part1/rayon01_01_rayonnage.jpg" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("rayonnage"),
+this
+),
+
+new CardObject(
+this,
+{ name: "panier", url: "sprites/StoreScene/part1/rayon01_02_panier.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("panier"),
+this
+),
+
+new CardObject(
+this,
+{ name: "liste", url: "sprites/StoreScene/part1/rayon01_04_liste.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("liste"),
+this
+),
+
+new CardObject(
+this,
+{ name: "pate09", url: "sprites/StoreScene/part1/rayon01_03_pate09.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("pate09"),
+this
+),
+
+new CardObject(
+this,
+{ name: "light", url: "sprites/StoreScene/part1/rayon01_00_model_light.jpg" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("light"),
+this
+),
+
+new CardObject(
+this,
+{ name: "model", url: "sprites/StoreScene/part1/rayon01_00_model.jpg" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("model"),
+this
+),
+
+new CardObject(
+this,
+{ name: "pate08", url: "sprites/StoreScene/part1/rayon01_03_pate08.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("pate08"),
+this
+),
+
+new CardObject(
+this,
+{ name: "rature", url: "sprites/StoreScene/part1/rayon01_05_rature.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("rature"),
+this
+),
+
+new CardObject(
+this,
+{ name: "farine08", url: "sprites/StoreScene/part1/rayon01_03_farine08.png" },
+new Phaser.Math.Vector2(0, -300),
+(scene) => scene.takeObject("farine08"),
+this
+),
+
```

Comment:
This sort of file shouldn't be on master.

Reply:
Removed – just an example for you